packages/cli/src/command-helpers/constants.ts

```diff
@@ -0,0 +1,29 @@
+export const PUBLIC_RPC_ENDPOINTS: any = {
```

Comment:
Although I like this pattern better than what we have in [`getEtherscanLikeAPIUrl`](https://github.com/graphprotocol/graph-tooling/blob/main/packages/cli/src/command-helpers/abi.ts#L58), I think consistency is key. So I suggest to either refactor `getEtherscanLikeAPIUrl` to use constants too.

Reply:
agreed I think what we have`getEtherscanLikeAPIUrl` is more safe

Reply:
@saihaj @schmidsi how do you propose going about this? Merge this as it is for now? And maybe migrate to a cleaner way in a future PR?

There are other places that could use some consistency, currently the list of available network is sort of hardcoded 
[here](https://github.com/graphprotocol/graph-tooling/blob/74bb29c7125930c7d0817861340ab9bed349517b/packages/cli/src/protocols/index.ts#L76).

If we could make this a single array with objects that has the etherscan URL and the public RPC endpoint all combined into a single object that is used everywhere where we need the list of supported networks, their etherscan/public RPC endpoint, that would be good in my opinion. Let me know what you folks think

Reply:
I think for this one let's just copy `getEtherScanLikeAPIUrl` way cause it is better than doing `any` type here. I do agree we need to clean this all up and have networks as a constant that we can use anywhere.